applications/crossbar/priv/couchdb/schemas/system_config.tasks.fax_cleanup.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "_id": "system_config.tasks.fax_cleanup",
+    "description": "Schema for tasks.fax_cleanup system_config",
+    "properties": {
+        "chunk_size": {
+            "default": 32,
+            "description": "The number of documents to process in one fetch operation",
+            "type": "integer"
+        },
+        "default_age_out": {
```

Comment:
I prefer to add the unit of time to the name of the variable - `default_age_out_s` and `per_chunk_pause_ms` in this case.